Nos/Views/SettingsView.swift

```diff
@@ -38,6 +39,7 @@ struct SettingsView: View {
                     if privateKeyString.isEmpty {
                         self.keyPair = nil
                         analytics.logout()
+                        appController.configureCurrentState()
```

Comment:
I like that this enables us to jump back out to onboarding. I don't love that when we complete onboarding again we land back on the settings page

Reply:
Good point. I think I can just clear the sideMenuPath and close the side menu right here. I will try that in my next PR.